### ISSUE TITLE
Frontend UI improvements

### ISSUE THREAD

**Issue by Jash271** [Jash271/E-Learning-Website#1]
Add transitions, animation, Bootstrap to enhance UI

**Comment by yashqwerty1** [comment]
Assign Me this. U want to work on it.

**Comment by Jash271** [comment]
> Assign Me this. U want to work on it.

Hey I went through your profile,you haven't done much of frontend,You're sure you'd be able to do this UI improvements,cause I am expecting some decent quality work ?

**Comment by yashqwerty1** [comment]
Yes Let me do some work and If it's not as your perfection. You can assign another person this work.

**Comment by Jash271** [comment]
> > Add transitions, animation, Bootstrap to enhance UI
> 
> @Jash271 can you please provide the `requirements.txt` file

Sure

**Comment by Jash271** [comment]
Hey @clare0901 ,whats the status with the frontend ?

**Comment by clare0901** [comment]
> Hey @clare0901 ,whats the status with the frontend ?

I have completed the following files,
login.html
registration_form.html
home.html
course_list.html
profile.html
edit_profile.html

Also the "enroll for this course"  link which is present in course_list.html doesn't work.
Will complete the rest of the files in following 3-4 days.
Thanks.

**Comment by clare0901** [comment]
> Hey @clare0901 ,whats the status with the frontend ?

Also I'm unable to access some parts of the website, such as 
1. I get an error when i try to `publish a course`
2. I get an error on `View Courses You have created`
3. After I submit the code, I can't see anything in `View Submissions`
4. Error on clicking `enroll for this course`

Also when is the `flag == 0` ? Because I can't add styling to `Create Course,Become a tutor Now`

Status up till now: 14/21 Pages are completed !!

**Comment by clare0901** [comment]
@Jash271
